pkg/shares/shares_test.go

```diff
@@ -0,0 +1,573 @@
+package types
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"math"
+	"math/rand"
+	"reflect"
+	"sort"
+	"testing"
+	"time"
+
+	"github.com/celestiaorg/rsmt2d"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/tendermint/tendermint/pkg/consts"
+	coretypes "github.com/tendermint/tendermint/types"
+)
+
+type Splitter interface {
+	SplitIntoShares() NamespacedShares
+}
+
+// func TestMakeShares(t *testing.T) {
```

Comment:
Why is this test commented?

Reply:
oh shoot, good point forgot about this. This test requires things like implementations of evidence and a "ShareSplitter" interface which is no longer followed. will create an issue to make sure we do the equivalent and manually checking the shares in the upcoming refactor

Reply:
#630